src/api/operations/units.fragments.graphql

```diff
@@ -28,4 +28,10 @@ fragment UnitFields on Unit {
       ...ClientName
     }
   }
+  latestOpportunity {
+    ...CeOpportunitySummaryFields
+    referral {
+      ...CeReferralSummaryFields
+    }
+  }
```

Comment:
This is the first time we'd have production users making API request that hit the new CE api types. It does "lock us in" a bit so we need to be more careful about not making backwards-incompatible changes.

Since CE isn't going to go live for a long while, it might be preferable to, add a directive here so we only query for it if CE is enabled. That way, if we did make a breaking change to this shape (like rename `latestOpportunity`, it wouldn't affect users.). Maybe I'm being overly cautious, it's also not a huge deal to start paying closer attention to backwards compatibility.

Reply:
Thanks, yeah I totally agree with this. Is https://github.com/greenriver/hmis-frontend/pull/1128/commits/b65071601c4a414c41a3dc4600bc52f37ce2c58d along the lines of what you had in mind?